src/Fortran/kittie.F90

```diff
@@ -5,8 +5,8 @@ module kittie
 
 
 	type coupling_helper
-		character(len=:), allocatable :: filename
-		character(len=:), allocatable :: groupname
+		character(len=512), allocatable :: filename
```

Comment:
Is there something you're doing that needs an explicit max length?

Reply:
NUTS, I think that got added by accident on Thursday and shouldn't be there. Sorry, I'll close this and resubmit.